src/common.js

```diff
@@ -38,3 +39,20 @@ export const inode_types = {
     link: _("Symbolic link"),
     special: _("Special file"),
 };
+
+export const allowedtoRead = (selected, user) => {
+    const readPermissions = (perm) => (parseInt(perm) & 4) === 4;
+    // superuser can read everything
+    if (superuser.allowed)
+        return true;
+
+    if (selected.owner === user.name && readPermissions(selected.permissions[0]))
+        return true;
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-175-20240129-100727-b86fcbeb-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)